manifests/01-namespace.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    openshift.io/cluster-logging: "true"
```

Comment:
should we add an empty node selector here? The daemonset doesn't play nice if there are additional node selector labels from the project (per @thoraxe )

Reply:
> should we add an empty node selector here? The daemonset doesn't play nice if there are additional node selector labels from the project (per @thoraxe )

Specify the same selector on your daemonset and you'll have no trouble.  And the controller should no longer destroy the cluster when it's denied (since 3.10 I think)

Reply:
@deads2k  You mean here [1]? or apply the same annotation to the Daemonset?

[1] https://github.com/openshift/cluster-logging-operator/blob/master/pkg/k8shandler/collection.go#L181

Reply:
@deads2k could that potentially be problematic though for customers? if the daemonset lives in the `openshift-logging` namespace and it has a default node selector, unless they have that node selector on all nodes, they'll have a gap in log collection...